src/index.html

```diff
@@ -5,9 +5,72 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card body__card" data-qa="card">
+      <div class="card__picture">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+      </div>
+
+      <div class="card__desription">
+        <h1 class="card__mark">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </h1>
+      </div>
+
+      <div class="card__productCode">
+        <p class="card__code">Product code: 195434</p>
+      </div>
+
+      <div class="card__rate">
+        <div class="stars">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star stars__star--nomargin"></div>
+        </div>
+
+        <div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+      </div>
+
+      <div class="card__value">
+        <div>
+          <p class="card__price">Price:</p>
+        </div>
+
+        <div>
+          <p class="card__money">$2,199</p>
+        </div>
+      </div>
+
+      <a
+        href="#buy"
+        class="card__link"
+        data-qa="hover"
+      >
+
+        <div class="card__buttom">
+          Buy
+        </div>
+      </a>
```

Comment:
```suggestion
 <div class="card__buttom">
      <a
        href="#buy"
        class="card__link"
        data-qa="hover"
      >
           Buy
       </a>
 </div>
```/* as for me this variant is more semantic than your */